source/CardGames.py

```diff
@@ -12,7 +12,26 @@ def find_root_dir():
     os.chdir('..')
     cwd = os.path.join( cwd, '..')
   return cwd
-
+def startUp():
```

Comment:
The startup looks good. Instructions are clear. I'd suggest changing the "Go Fish Card Game" title to "Welcome to Go-Fish!".

Reply:
Thanks, I'll definitely do that, it'll make it more welcoming 👍